File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/possible_change.java
--- 
+++ 
@@ -5,6 +5,9 @@
     if (total < 0) {
         return 0;
     }
+    if (coins.length == 0) {
+        return 0;
+    }
 
     int first = coins[0];
     int[] rest = Arrays.copyOfRange(coins, 1, coins.length);

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/codex-sh/possible_change.py
--- 
+++ 
@@ -1,7 +1,7 @@
 def possible_change(coins, total):
     if total == 0:
         return 1
-    if total < 0:
+    if total < 0 or not coins:
         return 0
 
     first, *rest = coins
